plugins/source/datadog/resources/plugin/tables.go

```diff
@@ -33,4 +72,12 @@ func Tables() []*schema.Table {
 		synthetics.Synthetics(),
 		users.Users(),
 	}
+	if err := transformers.TransformTables(tables); err != nil {
+		panic(err)
+	}
+	for _, table := range tables {
+		addCqIDs(table)
+		titleTransformer(table)
```

Comment:
@yevgenypats 
using this will construct the csr for each table, instead of constructing caser once & reusing it for all transformations, though